infrastructure/config.sh

```diff
@@ -18,7 +18,7 @@
 export aws_region="eu-west-1"
 export dynamo_table_name="terraform-lock"
 export bucket_name="terraform-state-xc3"
-export project="example"
-export domain="example.co"
-export owner_email="admin@example.co"
-export creator_email="admin@example.co"
+export project="xgrid"
```

Comment:
Is this expected behavior to add information of xgrid?

Reply:
Added in a5d15f73de7421283d2c3bb516e4479656be7cf8